server/chat-commands/core.ts

```diff
@@ -490,7 +490,7 @@ export const commands: ChatCommands = {
 		if (!targetId || !message) {
 			return this.parse(`/help msgroom`);
 		}
-		const targetRoom = Rooms.search(toID(targetId));
+		const targetRoom = Rooms.search(("" + targetId).toLowerCase().replace(/[^a-z0-9-]+/g, ''));
```

Comment:
Wait, why is this done at all?

Reply:
because toID replaces '-', which means we can't message the chess game.